cypress_shared/pages/assess/show.ts

```diff
@@ -52,4 +67,36 @@ export default class AssessmentShowPage extends Page {
       })
     })
   }
+
+  shouldShowCreateNoteForm() {
+    cy.get('.referral-note')
+      .contains('Add a note')
+      .parent()
+      .parent()
+      .within(() => {
+        cy.get('.govuk-textarea').should('be.visible')
+        cy.get('.govuk-button--secondary').should('be.visible')
+      })
+  }
+
+  createNote(message: string) {
+    cy.get('.referral-note')
+      .contains('Add a note')
+      .parent()
+      .parent()
+      .within(() => {
+        cy.get('.govuk-textarea').type(message)
```

Comment:
Think this could be a call down to `completeTextInputByLabel` in the base class - `completeTextInputByLabel('Add a note')`. And getting more into personal preference, consider just doing something like `cy.get('button').contains('Save note').click()`

Reply:
:recycle: